airbyte/_executors/python.py

```diff
@@ -65,11 +65,13 @@ def __init__(
             if metadata and metadata.pypi_package_name
             else f"airbyte-{self.name}"
         )
-        self.install_root = install_root or Path.cwd()
+        self.install_root = install_root or DEFAULT_INSTALL_DIR or Path.cwd()
+        with suppress(Exception):
+            self.install_root.mkdir(parents=True, exist_ok=True)
         self.use_python = use_python
 
     def _get_venv_name(self) -> str:
-        return f".venv-{self.name}"
+        return f"{DEFAULT_VENV_PREFIX}-{self.name}"
```

Comment:
This doesn't add much. Lt's revert the DEFAULT_VENV_PREFIX constant and just use the prev.


```suggestion
        return f".venv-{self.name}"
```